<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1315</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1315</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Torrent Aggregator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="search_term">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Term</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="manager">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>1071</width>
      <height>421</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Seeds</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Leechs</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Info</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Web Site</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="next_page_btn">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>450</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_page_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>209</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="check_box_pirate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>The Pirate Bay</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_box_kickass">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kickass Torrents</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_box_isohunt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Isohunt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>52</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Websites</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
